crates/genie-hki/src/lib.rs

```diff
@@ -1,9 +1,15 @@
 //! Reader/writer for Age of Empires 2 Hotkey info files.
 //!
-//! Hotkey files in AoE2 contain groups, each of which contain some number of hotkeys. Hotkeys have
-//! a string ID, a keycode, and flags for Ctrl/Alt/Shift modifiers. The index of the hotkey in its
+//! Hotkey files in AoE2 contain groups, each of which contain some number of
+//! hotkeys. Hotkeys have a string ID, a keycode, and flags
+//! for Ctrl/Alt/Shift modifiers. The index of the hotkey in its
 //! group determines the action that will be taken when it is activated.
-use std::io::{Read, Write, Result};
+
+use std::error::Error;
+use std::io;
+use std::io::{Read, Write};
```

Comment:
nit: `use std::io::{self, Read, Write};` will import the Read and Write traits as well as the io namespace.

Reply:
ty!